zerver/lib/avatar.py

```diff
@@ -147,3 +147,7 @@ def is_avatar_new(ldap_avatar: bytes, user_profile: UserProfile) -> bool:
         return False
 
     return True
+
+
+def get_avatar_for_inaccessible_user() -> str:
+    return staticfiles_storage.url("images/default-avatar.png")
```

Comment:
I have used this avatar just for now, as we had not decided the color before. Will update it soon.

Reply:
OK